akka-docs/src/main/paradox/stream/operators/Source/unfoldResource.md

```diff
@@ -14,14 +14,51 @@ Wrap any resource that can be opened, queried for next element (in a blocking wa
 
 ## Description
 
-Wrap any resource that can be opened, queried for next element (in a blocking way) and closed using three distinct functions into a source.
+`Source.unfoldResource` allows us to safely interact with this API by providing three functions: 
```

Comment:
What is "this API"?

Reply:
That API, you know! Not sure what happened in this sentence, I'll fix.